packages/babel-parser/test/fixtures/experimental/dynamic-import/direct-calls-only/output.json

```diff
@@ -0,0 +1,172 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 49,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 3,
+      "column": 1
+    }
+  },
+  "errors": [
+    "SyntaxError: Dynamic imports require a parameter: import('a.js') (2:9)",
+    "SyntaxError: The only valid meta property for import is import.meta (2:16)"
```

Comment:
I like this double error, since it hints about the two possible interpretations.

Input code:
```js
function failsParse() {
  return import.then();
}
```